openmdao/vectors/vector.py

```diff
@@ -102,7 +102,6 @@ def __init__(self, name, kind, system, root_vector=None, alloc_complex=False):
         self._name = name
         self._typ = _type_map[kind]
         self._kind = kind
-        self._icol = None
```

Comment:
need to remove class docstring entry for `_icol`   
:docstring_police:

Reply:
done